composer.json

```diff
@@ -1,44 +1,46 @@
-{
-  "name": "opulence/net",
-  "description": "The Opulence network library",
-  "keywords": [
-    "PHP",
-    "network",
-    "http",
-    "request",
-    "response",
-    "cookie"
-  ],
-  "authors": [
-    {
-      "name": "David Young",
-      "email": "dave@opulencephp.com",
-      "role": "Architect"
-    }
-  ],
-  "license": "MIT",
-  "autoload": {
-    "psr-4": {
-      "Opulence\\Net\\": "src"
-    },
-    "exclude-from-classmap": [
-      "**/Tests/"
-    ]
-  },
-  "require": {
-    "ext-mbstring": "*",
-    "opulence/collections": "1.1.*@dev",
-    "opulence/io": "1.1.*@dev",
-    "opulence/serialization": "1.0.*@dev",
-    "php": "^7.2"
-  },
-  "require-dev": {
-    "phpunit/phpunit": "~7.3"
-  },
-  "extra": {
-    "branch-alias": {
-      "dev-master": "1.0.x-dev"
-    }
-  },
-  "minimum-stability": "dev"
-}
\ No newline at end of file
+{
+  "name": "opulence/net",
+  "description": "The Opulence network library",
+  "keywords": [
+    "PHP",
+    "network",
+    "http",
+    "request",
+    "response",
+    "cookie"
+  ],
+  "authors": [
+    {
+      "name": "David Young",
+      "email": "dave@opulencephp.com",
+      "role": "Architect"
+    }
+  ],
+  "license": "MIT",
+  "autoload": {
+    "psr-4": {
+      "Opulence\\Net\\": "src"
+    }
+  },
+  "autoload-dev": {
```

Comment:
This probably isn't necessary anymore now that we moved the tests back under `src/Tests`

Reply:
I think this block should be existed because the namespace is different.

Reply:
The test namespace will be resolved by the `autoload` psr-4 config because the tests are in the `src/Tests` directory.

Reply:
@opulencephp, thank you for your reply. You're right and I remove the `autoload-dev` block in latest commit now :-).